metrics.md

```diff
@@ -8,39 +8,36 @@ You may obtain a copy of the License at
     http://www.apache.org/licenses/LICENSE-2.0
 -->
 
-## Metrics
+# Metrics
 
-Pravega metrics are collected and exposed to Flink metrics framework when using `FlinkPravegaReader` or `FlinkPravegaWriter` instances to perform read and write operations on Pravega streams.
+Pravega metrics are collected and exposed to Flink metrics framework when using `FlinkPravegaReader` or `FlinkPravegaWriter` instances to perform read and write operations on Pravega Streams.
 
-### Reader Metrics
+## Reader Metrics
 
-The following metrics are exposed and made available as Flink metrics for `FlinkPravegaReader` related operations.
-
-- reader group name
-- scope
-- stream names
-- online readers
-- segment positions
-- unread bytes
+The following metrics are exposed and made available as Flink metrics for `FlinkPravegaReader` related operations:
 
 Metric Name                |Description|
 |-----------------|-----------------------------------------------------------------------|
-|`readerGroupName`|The name of the reader group.|
-|`scope`|The scope name of the reader group.|
-|`streams`|The streams that are part of the reader group. The stream name will be a fully qualified name i.e., `scope/stream`|
+|`readerGroupName`|The name of the ReaderGroup.|
+|`scope`|The scope name of the ReaderGroup.|
+|`streams`|The streams that are part of the ReaderGroup. The stream name will be a fully qualified name i.e., `scope/stream`|
 |`onlineReaders`|The readers from the reader group that are currently online/available.|
-|`segmentPositions`|The StreamCut information that indicates where the readers have read so far.|
+|`segmentPositions`|The `StreamCut` information that indicates where the readers have read so far.|
 |`unreadBytes`|The total number of bytes that have not been read yet.|
 
-### Writer Metrics
+## Writer Metrics
+
+For `FlinkPravegaWriter` related operations, only the stream name is exposed:
 
-For `FlinkPravegaWriter` related operations, only the stream name is exposed.
+Metric Name                 |Description|
+|-----------------|-----------------------------------------------------------------------|
+|`streams`  |The streams that are part of the ReaderGroup. The stream name will be a fully qualified name i.e., `scope/stream`|

```

Comment:
The column width for the "Metric Name" is less. Please use the same column width from the pervious section.

Reply:
markdown columns width is determined by the longest cell in the column,